xml/favicons.xml

```diff
@@ -121,7 +121,15 @@
     <param pos="0" name="service.product" value="XAMPP Server"/>
     <param pos="0" name="service.certainty" value="0.5"/>
   </fingerprint>
-
+  
+ <fingerprint pattern="^bdcedebae3582b73c452f57bf05b1345$">
+    <description>Medecom Clipper</description>
+    <example>bdcedebae3582b73c452f57bf05b1345</example>
+    <param pos="0" name="service.vendor" value="Medecom"/>
+    <param pos="0" name="service.product" value="Clipper"/>
+    <param pos="0" name="service.certainty" value="0.5 "/>
```

Comment:
Do you have anything to support only providing 0.5 certainty that this hash is the given service? This is a parameter I tend to not copy from some of the other fingerprints in this file.